README.md

```diff
@@ -73,24 +60,48 @@ This cookbook makes no assumption about the webserver you use.
 Therefore this is *your* responsability to write a wrapper and notify your
 webserver (ie. Apache or php5-fpm) for reload/restart or subscribe the good
 resources.
+To do that, you can use `blackfire_php` resource to manage blackfire.ini and notify php service according to php version. It's very usefull if you have many php version on the same server.
```

Comment:
Correct me if I'm wrong but I don't see any service notification.
But I think it would be hard to achieve so I believe this part should be removed.

Reply:
Like you said above in README, the notification must be done in a wrapper cookbook.
I will update the README in the resource part to add `notifies` parameter example to notify PHP service (or HTTP service) which has been previously installed (indeed).

Reply:
Readme updated. Let me know if it's OK for you